### PR TITLE
fix pegasushoof caps light, add via keymap

### PR DESCRIPTION
**Pull request by wangfuco**
## Description
Regarding the pegasus hoof keyboard, the leds were misconfigured and didn't work. I fixed it and verified the fix.

I also added a via keymap based on the default layout so it should support the ansi and iso variants.

The work is heavily based on [https://github.com/Tecsmith/flying-horse-foot](url)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bpiphany/pegasushoof/2013/2013.h [hunk 1]
```diff
@@ -74,8 +74,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 }
 
-inline void ph_caps_led_on(void)  { DDRC |=  (1<<6); PORTC &= ~(1<<6); }
-inline void ph_caps_led_off(void) { DDRC &= ~(1<<6); PORTC &= ~(1<<6); }
+inline void ph_caps_led_on(void)  { PORTC &= ~(1<<6); }
+inline void ph_caps_led_off(void) { PORTC |=  (1<<6); }
 
-inline void ph_sclk_led_on(void)  { DDRC |=  (1<<5); PORTC &= ~(1<<5); }
-inline void ph_sclk_led_off(void) { DDRC &= ~(1<<5); PORTC &= ~(1<<5); }
+inline void ph_sclk_led_on(void)  { PORTC &= ~(1<<5); }
```
Comment (waffle87) [inline keyboards/bpiphany/pegasushoof/2013/2013.h:80]: All these `ph_sclk_led_*()` functions should be replaced with [indicator pin defines](https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options).
They should go in `info.json` file(s) as 
```json
"indicators": {
    "caps_lock" : "C6",
    "scroll_lock": "C5",
}
```

File: keyboards/bpiphany/pegasushoof/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    ),
+
+    /* Layer 2: Additional layer */
+    [LAYER_3] = LAYOUT( \
+        QK_BOOT,          XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX, \
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX, \
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX, \
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          DB_TOGG,                                 \
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          XXXXXXX,             XXXXXXX,          \
+        XXXXXXX, XXXXXXX, XXXXXXX,                            XXXXXXX,                            XXXXXXX, _______, _______, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX  \
```
Comment (waffle87) [inline keyboards/bpiphany/pegasushoof/keymaps/via/keymap.c:54]:
```suggestion
    [LAYER_1] = LAYOUT(
        KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,     KC_PSCR, KC_SCRL, KC_PAUS,
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,    KC_INS,  KC_HOME, KC_PGUP,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,    KC_DEL,  KC_END,  KC_PGDN,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,             KC_UP,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, MO(1),   KC_RCTL,    KC_LEFT, KC_DOWN, KC_RGHT
    ),

    /* Layer 1: Function layer */
    [LAYER_2] = LAYOUT(
        _______,          KC_BRID, KC_BRIU, KC_F16,  KC_F17,  KC_F18,  KC_F19,  KC_MPRV, KC_MPLY, KC_MNXT, KC_MUTE, KC_VOLD, KC_VOLU,    _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_ERAS,    _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_EJCT,    _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          KC_PENT,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,             _______,
        _______, _______, _______,                            _______,                            _______, MO(2),   _______, _______,    _______, _______, _______
    ),

    /* Layer 2: Additional layer */
    [LAYER_3] = LAYOUT(
        QK_BOOT,          XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX,
        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX,
        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX,
        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          DB_TOGG,
        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          XXXXXXX,             XXXXXXX,
        XXXXXXX, XXXXXXX, XXXXXXX,                            XXXXXXX,                            XXXXXXX, _______, _______, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX
```

File: keyboards/bpiphany/pegasushoof/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    } else {
+        ph_caps_led_off();
+    }
+
+    if (usb_led & (1 << USB_LED_SCROLL_LOCK)) {
+        ph_sclk_led_on();
+    } else {
+        ph_sclk_led_off();
+    }
+}
```
Comment (waffle87) [inline keyboards/bpiphany/pegasushoof/keymaps/via/keymap.c:71]: Above comment w.r.t. indicators.
```suggestion
```

File: keyboards/bpiphany/pegasushoof/keymaps/via/rules.mk [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+CUSTOM_MATRIX = yes         # Custom matrix file for the Pegasus Hoof due to the 2x74HC42
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = no        # RGB underlight.
+
```
Comment (waffle87) [inline keyboards/bpiphany/pegasushoof/keymaps/via/rules.mk:15]:
```suggestion
MOUSEKEY_ENABLE = no        # Mouse keys
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/bpiphany/pegasushoof/2013/2013.h (hunk 1 above)
Comment (wangfuco) [reply keyboards/bpiphany/pegasushoof/2013/2013.h:80]: Thank you for the suggestion! I'll implement these changes.

**Comment by wangfuco** [comment]
@waffle87 All your suggestions should be in place now!

**Review by waffle87 — APPROVED** [review]

File: keyboards/bpiphany/pegasushoof/keymaps/via/rules.mk [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+MOUSEKEY_ENABLE = no        # Mouse keys
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+
```
Comment (fauxpark) [inline keyboards/bpiphany/pegasushoof/keymaps/via/rules.mk:8]:
```suggestion
MOUSEKEY_ENABLE = no
CONSOLE_ENABLE = no
COMMAND_ENABLE = no
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by wangfuco** [comment]
@fauxpark Please let me know if there are any pending items!

**Review by fauxpark — APPROVED** [review]

**Pull request merged**